common/lib/xmodule/xmodule/course_module.py

```diff
@@ -908,7 +909,8 @@ class CourseFields(object):
             "This setting only applies if proctored exams are enabled for the course."
         ),
         default=True,
-        scope=Scope.settings
+        scope=Scope.settings,
+        deprecated=EXAM_SETTINGS_HTML_VIEW_ENABLED
```

Comment:
Just in case you are looking for a testing file to add to or modify, the one I created was cms/djangoapps/contentstore/views/tests/test_exam_settings_view.py, from this [PR](https://github.com/edx/edx-platform/pull/24405).